inst/components/scss/sidebar.scss

```
@@ -60,7 +61,8 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--bslib-sidebar-icon-size)),
     > .sidebar-content {
       display: flex;
       flex-direction: column;
-      padding: var(--bslib-sidebar-padding);
+      gap: var(--bslib-spacer, 1rem);
+      padding: var(--bslib-sidebar-content-padding);
```

Assuming we go the direction of https://github.com/rstudio/bslib/pull/725/files#r1284825876

```suggestion
      padding: var(--bslib-sidebar-padding);
```